### PR TITLE
Add documentation for the code repair and transpile tasks

### PR DESCRIPTION
**Pull request by ruiAzevedo19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,7 +209,16 @@ It needs to compile and provide 100% coverage. A model can only write such tests
...
+Currently, the following cases are available for this task:
+
+- Java
+  - `plain/src/main/java/plain.java`: An empty function that does nothing.
+- Go
+  - `plain/plain.go`: An empty function that does nothing.
+
+### Reward Points
 
 Currently, the following points are awarded for this task:
```
Comment (bauersimon) [inline README.md:254]: `Currently, the following points are awarded for any task:`

also could you please change the coverage scoring below... it is not based on 100% anymore but what portions of the code is covered

File: README.md [hunk 2]
```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
 - Go
   - `plain/plain.go`: An empty function that does nothing.
 
+### Task: Code Repair
+
+Code repair is the task of repairing source code with compilation errors.
+
+For this task, we introduced the `mistakes` repository, which includes five examples of source code with compilation errors for both Java and Go.
```
Comment (bauersimon) [inline README.md:225]: Would remove this sentence cause the number of examples and languages could change.

File: README.md [hunk 3]
```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
 - Go
   - `plain/plain.go`: An empty function that does nothing.
```
Comment (bauersimon) [inline README.md]: We have more than the plain examples by now. Please add a commit where they are added `with pretty links`.

File: README.md [hunk 4]
```diff
@@ -218,6 +218,22 @@ Currently, the following cases are available for this task:
...
+Each example is isolated in its own package, along with a valid test suite. We compile each package in the `mistakes` repository and provide the LLM's with both the source code and the list of compilation errors. The LLM's response is then validated with the predefined test suite.
+
+#### Cases
+
+Currently, the following cases are available for this task:
+
+- Java
+  - [`java/mistakes`](https://github.com/symflower/eval-dev-quality/tree/main/testdata/java/mistakes)
+- Go
+  - [`golang/mistakes`](https://github.com/symflower/eval-dev-quality/tree/main/testdata/golang/mistakes)
```
Comment (bauersimon) [inline README.md]: Can you try to change the link to be relative? I.e. `[java/mistakes](testdata/java/mistakes)` cause I think GitHub will automatically make it a link to the folder. That way it stays correct on different branches.

**Review by bauersimon — CHANGES_REQUESTED** [review]

**Comment by bauersimon** [comment]
awesome! will merge after https://github.com/symflower/eval-dev-quality/pull/226 is in

**Comment by ruiAzevedo19** [comment]
Closing in favor of #264 and #226

**Pull request closed without merging**